Maple2.Server.Game/Model/Field/Actor/FieldNpc.cs

```diff
@@ -210,6 +215,12 @@ public class FieldNpc : Actor<Npc> {
             foreach (string tag in Value.Metadata.Basic.MainTags) {
                 player.Session.ConditionUpdate(ConditionType.npc_race, codeString: tag);
             }
-        }
+		}
+	}
+	
+    public void SetAi(string name) {
+        AiMetadata? metadata;
+
+        Field.AiMetadata.TryGet(name, out metadata);
```

Comment:
this func is currently not doing setting the metadata
```suggestion
        if (Field.AiMetadata.TryGet(name, out AIMetadata? metadata)) {
            AiMetadata = metadata;
        };
```